.github/workflows/setup-helper.yml

```
@@ -0,0 +1,93 @@
+      uses: actions/checkout@v4
+    
+    - name: Generate .env file from secrets
+      run: |
+        cat > .env << EOF
+        # Database Configuration
+        DB_HOST=localhost
+        DB_PORT=5432
+        DB_NAME=compass
+        DB_USERNAME=compass_user
+        DB_PASSWORD=compass_password
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)